package.json

```diff
@@ -13,7 +13,7 @@
   },
   "homepage": "https://github.com/endpass/endpass-auth",
   "scripts": {
-    "dev": "SOURCE_MAP=true vue-cli-service serve",
+    "dev": "vue-cli-service serve",
```

Comment:
what is the reason to remove `SOURCE_MAP=true` ?

Reply:
collision, locally I forced to remove it, because we don't use crossenv